sonic_platform_base/module_base.py

```diff
@@ -264,7 +264,7 @@ def get_state_info(self):
 
     def get_bus_info(self, module_name):
```

Comment:
Do we need the module_name as a parameter for these functions? The method would be part of the `Module` class which is implemented, which means that `get_bus_info` `get_pci_reattach` and `get_pci_detach` would be executed for specific modules

Reply:
good point. Fixed it. Thank you!